subprojects/docs/src/docs/userguide/repositoryTypes.adoc

```diff
@@ -181,6 +181,60 @@ You can specify credentials for Ivy repositories secured by basic authentication
 </sample>
 ++++
 
+[[sub:supported_metadata_sources]]
+=== Supported metadata sources
```

Comment:
We probably don't need all the "for example"s throughout this section. Let's keep our docs [Strunk & White](https://en.wikipedia.org/wiki/The_Elements_of_Style).

Reply:
There was indeed a duplicated sentence here. I missed that. And I think that gave the wrong impression. There is only one example here, and I think it's good to have one.